ext/curl/interface.c

```diff
@@ -2070,49 +2186,41 @@ static inline int build_mime_structure_from_hash(php_curl *ch, zval *zpostfields
 }
 /* }}} */
 
-/* {{{ proto resource curl_copy_handle(resource ch)
+/* {{{ proto Curl curl_copy_handle(CurlHandle ch)
```

Comment:
```suggestion
/* {{{ proto CurlHandle curl_copy_handle(CurlHandle ch)
```

I know how much you love the protos!

Reply:
:D Very much :)